composer.json

```diff
@@ -65,19 +18,23 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "php": "^7.4",
+        "ext-pdo": "*",
+        "psr/log": "^1.1",
         "yiisoft/arrays": "^3.0@dev",
+        "yiisoft/cache": "^3.0@dev",
+        "yiisoft/log": "^3.0@dev",
```

Comment:
Doesn't seem to be used.

Reply:
Its used, for test, verify cache, logger, profiler work fine.